src/index.scss

```diff
@@ -299,6 +299,18 @@ svg.svg-wrapper {
     padding: 0.5rem 1rem;
   }
 
+  .main-nav nav.navbar .navbar-brand {
+    padding-left: 1rem;
+  }
+
+  .main-nav nav.navbar {
+    background-color: #000;
+  }
+
+  .navigation-search .version-wrapper:first-child {
+    padding: 0;
+  }
+
```

Comment:
Actually, I vote for removing mobile support from Desktop wallet.

Reply:
I agree. This code came from hathor-explorer and is not important here. I don't think we should remove support, but we don't have to worry about it.

Reply:
I'm confused.

> I agree

> I don't think we should remove support

Do you agree to remove the responsive code from the wallet? Given the bug it caused (see linked issue), I don't think we can just ignore it.

Reply:
Ok, let's keep mobile support